example.env

```diff
@@ -26,6 +26,7 @@ TM_ORG_FB=https://www.facebook.com/example
 TM_ORG_INSTAGRAM=https://www.instagram.com/example/
 TM_ORG_YOUTUBE=https://www.youtube.com/user/example
 TM_ORG_GITHUB=https://github.com/example/
+# TM_HOMEPAGE_VIDEO_URL=
```

Comment:
This is already in the `example.env` (around 20 lines further down) together with its description text

Reply:
sorry didn't notice, thanks for fixing